justfile

```diff
@@ -87,5 +87,9 @@ update-modern-normalize:
     https://raw.githubusercontent.com/sindresorhus/modern-normalize/main/modern-normalize.css \
     > static/modern-normalize.css
 
-graph:
-  ./bin/graph
+download-log host="ordinals.com":
+  ssh root@{{host}} 'journalctl -u ord > ord.log'
+  rsync --progress root@{{host}}:ord.log ord.log
+
+graph host="ordinals.com":
```

Comment:
Shouldn't this depend on download-log ?

Reply:
It could, although download-log is actually pretty slow. It takes around 3 minutes to download the current log from ordinals.com, so you probably don't want to re-download it every time you view the graph.

Reply:
Ah okay. Other than that does `host` need to be passed in to `graph`? It isn't used anywhere currently

Reply:
Good catch! Removed it.